seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/config/CommonConfig.java

```diff
@@ -80,6 +80,25 @@ public class CommonConfig implements Serializable {
                     .defaultValue(false)
                     .withDescription(" the iceberg case_sensitive");
 
+    // for kerberos
+    public static final Option<String> KERBEROS_PRINCIPAL =
+            Options.key("kerberos_principal")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("jdbc kerberos_principal");
+
+    public static final Option<String> KERBEROS_KEYTAB_PATH =
+            Options.key("kerberos_keytab_path")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("jdbc kerberos_keytab_path");
+
+    public static final Option<String> KERBEROS_KRB5_CONF_PATH =
+            Options.key("kerberos_krb5_conf_path")
+                    .stringType()
+                    .noDefaultValue()
+                    .withDescription("jdbc kerberos_keytab_path");
```

Comment:
How about use same name of kerberos config in https://github.com/apache/seatunnel/blob/16950a67cd5990ed3510fd91564390e2ab2cd44c/seatunnel-connectors-v2/connector-file/connector-file-base/src/main/java/org/apache/seatunnel/connectors/seatunnel/file/config/BaseSinkConfig.java#L231-L248 ?

Reply:
They belong to different connectors. If I want to abstract them into a public configuration class, I can do related work.

Reply:
> abstract them into a public configuration class

Yes please. Put it into https://github.com/apache/seatunnel/tree/16950a67cd5990ed3510fd91564390e2ab2cd44c/seatunnel-api/src/main/java/org/apache/seatunnel/api with new kerberos directory.

Reply:
I have resolved it